docs/scos/dev/front-end-development/202212.0/oryx/getting-started/oryx-environment.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Environment Variables"
+---
+
+Environment variables are useful when:
+
+- Values are different across developer machines.
+- Values are different across multiple environments: (dev, staging, qa, prod)
+- Values change frequently and are highly dynamic.
+
+Environment variables can be changed easily - especially when running in CI.
+
+Oryx uses [dotenv](https://github.com/motdotla/dotenv) for environment variables management. 
+
+## Storefront Variables 
+
+The list of environment variables to configure Storefront:
+
+| Variable | Required | Default | Description                               |
+|----------|----------|---------|-------------------------------------------|
+| `SCOS_BASE_URL` | Yes | None | Spryker GLUE API URL |
+| `STORE` | Yes | None | Default store name, e.g. DE, AT etc. The value is case-sensitive |
+| `PRICE_MODE` | No | None | Can be used to initialise the application with default price mode. When this variable is not provided, the backend will set the default one |
+| `ORYX_PRESET` | No | `b2c` | Used to set a preset name, available options: `b2b`, `b2c` |
+| `ORYX_FEATURE_VERSION` | No | `latest` | Used to set a feature level, available options: `latest`, `1.0`, `1.1`, etc. Only major and minor versions can be used |
+| `ORYX_LABS` | No | `false` | Turns on/off oryx labs features. These features are in experimental state and are not production ready |
+
+### Labs Variables
+
+There are some additional variables, which are available only when [labs feature](TODO: ADD LINK) is turned on:
```

Comment:
let's add the link

Reply:
Which link should we use here?  I didn't manage to find labs description in published files, only some links to npm packages

Reply:
fair! There's no good page with reasonable content to link to. We could link to https://www.npmjs.com/package/@spryker-oryx/labs for now.

Reply:
We have removed labs info completely, so it is not relevant anymore